dataworks-aws-ingest-reconcilers.tf

```diff
@@ -0,0 +1,80 @@
+resource "github_repository" "dataworks_aws_ingest_reconcilers" {
+  name             = "dataworks-aws-ingest-reconcilers"
+  description      = "Manages the reconcilers and trimmers for aws ingestion feeds"
+  auto_init        = false
+
+  allow_merge_commit     = false
+  delete_branch_on_merge = true
+  has_issues             = true
+  topics                 = concat(local.common_topics, local.aws_topics)
+
+  lifecycle {
+    prevent_destroy = true
+  }
+
+  template {
+    owner = var.github_organization
+    repository = "dataworks-repo-template-terraform"
+  }
+}
+
+resource "github_team_repository" "dataworks-aws-ingest-reconcilers_dataworks" {
+  repository = github_repository.dataworks_aws_ingest_reconcilers.name
+  team_id    = github_team.dataworks.id
+  permission = "push"
+}
+
+resource "github_branch_protection" "dataworks-aws-ingest-reconcilers_master" {
+  branch         = github_repository.dataworks_aws_ingest_reconcilers.default_branch
+  repository     = github_repository.dataworks_aws_ingest_reconcilers.name
+  enforce_admins = false
+
+  required_status_checks {
+    strict = true
+    contexts = ["concourse-ci/status"]
+  }
+
+  required_pull_request_reviews {
+    dismiss_stale_reviews      = true
+    require_code_owner_reviews = true
+  }
+}
+
+resource "github_issue_label" "dataworks-aws-ingest-reconcilers" {
+  for_each   = { for common_label in local.common_labels : common_label.name => common_label }
+  color      = each.value.colour
+  name       = each.value.name
+  repository = github_repository.dataworks_aws_ingest_reconcilers.name
+}
+
+resource "null_resource" "dataworks-aws-ingest-reconcilers" {
+  triggers = {
+    repo = github_repository.dataworks_aws_ingest_reconcilers.name
+  }
+  provisioner "local-exec" {
+    command = "./initial-commit.sh ${github_repository.dataworks_aws_ingest_reconcilers.name} '${github_repository.dataworks_aws_ingest_reconcilers.description}' ${github_repository.dataworks_aws_ingest_reconcilers.template.0.repository}"
+  }
+}
+
+locals {
+  dataworks_aws_ingest_reconcilers_pipeline_name = "aws-ingest-reconcilers"
```

Comment:
probably want to drop the `aws-` prefix

Reply:
With aws it matches our pattern more (See new concourse)?